DemoApp/android/build.gradle

```diff
@@ -30,6 +30,7 @@ allprojects {
         maven {
             url  "https://dl.bintray.com/vsappcenter/appcenter-snapshot"
         }
+        maven { url 'https://www.jitpack.io' }
```

Comment:
I see no new deps, so I doubt that we need it

Reply:
Removed this.